annotators/property_extraction/server.py

```diff
@@ -154,6 +154,12 @@ def generate_triplets(uttr_batch, relations_pred_batch):
     return triplets_corr_batch
 
 
+def is_question(uttr: str):
+    uttr = uttr.lower()
+    is_q = any([uttr.startswith(q_word) for q_word in ["what ", "who ", "when ", "where "]]) or "?" in uttr
```

Comment:
we have the same function to use in common (like is_any_question - i do not remember the title clearly but try to find it and use it)

Reply:
there is only is_questions that checks for question marks or another for factoid-only questions in `common.utils.py`

Reply:
not repeating templates you still can do the following:
`is_yes({"text": sentence, "annotations": {}})`